### PR TITLE
Map output fix

### PR DESCRIPTION
**Pull request by PrateekGupta1509**
- Fixed the map not working issue. (The problem was chrome.runtime.sendmessage doesn't work if options or know more page is open so the observer had to be initialized outside it).
- Added default CO2 emission stat in options page.
- Improvement in tree computed (rounding at initial stage led to loss of data in terms of month and days. Eg tree: 0.5 per year, rounding up made it zero, however it can be expressed as 1 tree for 6 months or 180 trees for a day).
- Error Handling and code cleanup.

### CODE REVIEW

File: Source/pages/js/utils.js [hunk 1]
```diff
@@ -1,51 +0,0 @@
-/**
```
Comment (Ceilican) [inline Source/pages/js/utils.js]: Was this file dead code only?

**Comment by Ceilican** [comment]
Many thanks!

File: Source/pages/js/utils.js (hunk 1 above)
Comment (PrateekGupta1509) [inline Source/pages/js/utils.js]: Yes. This file was part of the first refactor done for options page and however with the later changes made this file is no longer in use.

Comment (PrateekGupta1509) [inline Source/pages/js/utils.js]: Yes. This file was part of the first refactor done for options page, however with the later changes made this file is no longer in use.

**Pull request merged**